dali/python/nvidia/dali/external_source.py

```diff
@@ -397,6 +474,29 @@ def __call__(self, *, source = None, cycle = None, name = None, layout = None, c
         if batch is None:
             batch = True
 
+        if prefetch_queue_depth is None:
+            prefetch_queue_depth = self._prefetch_queue_depth or 1
+
+        if parallel is None:
```

Comment:
Same.

Reply:
As above, this is pattern for checking how it was specified once.